pytransform3d/plot_utils/_artists.pyi

```diff
@@ -53,3 +53,23 @@ class Arrow3D(FancyArrowPatch):
     def set_data(self, xs: npt.ArrayLike, ys: npt.ArrayLike, zs: npt.ArrayLike): ...
 
     def draw(self, renderer: RendererBase): ...
+
+class Camera(artist.Artist):
+    def __init__(
+        self, M: npt.ArrayLike, A2B: npt.ArrayLike,
```

Comment:
Could you change the name of A2B to cam2world? Or cam2origin? Here and in the .py file of course.